Update README.md

nolanpro opened the pull request:
Second CI Instance....

ci:deploy

```
core -> pm-apps
collection -> pm-apps
data-sources -> pm-apps
private-equity -> main

frontend:
pm-apps -> main

php artisan package-private-equity:install
php artisan package-ai:install

php artisan migrate
php artisan upgrade
```
ci:package-private-equity:epic/FOUR-24127
ci:package-collections:epic/FOUR-24127
ci:package-data-sources:epic/FOUR-24127
ci:connector-send-email:epic/FOUR-24127
ci:modeler:epic/FOUR-24127
ci:package-ai:epic/FOUR-24127

ci:k8s-branch:pe

pmPaulis committed:
FOUR-22837

pmPaulis committed:
Merge pull request #8160 from ProcessMaker/feature/FOUR-22837

FOUR-22837:  Enable in the .env the configuration related session.same_site / session.cookie_secure

marcoAntonioNina committed:
Merge branch 'release-2025-spring' of github.com:ProcessMaker/processmaker into release-2025-spring

fagubla committed:
FOUR-23362:S1: Add in the setting the menu Iframe Whitelist Config

fagubla committed:
Update Model Setting

fagubla committed:
CR

fagubla committed:
add model in upgrade

pmPaulis committed:
Merge pull request #8163 from ProcessMaker/feature/FOUR-23362

FOUR-23362:S1 : Add in the setting the menu Iframe Whitelist Config

pmPaulis committed:
FOUR-23128

marcoAntonioNina committed:
Merge branch 'release-2025-spring' of github.com:ProcessMaker/processmaker into release-2025-spring

pmPaulis committed:
Update the key

pmPaulis committed:
Merge pull request #8171 from ProcessMaker/feature/FOUR-23128

FOUR-23128: Include the links saved in Settings

marcoAntonioNina committed:
Merge branch 'release-2025-spring' of github.com:ProcessMaker/processmaker into release-2025-spring

pmPaulis committed:
FOUR-23765

pmPaulis committed:
FOUR-23805

pmPaulis committed:
Merge pull request #8180 from ProcessMaker/feature/FOUR-23765

FOUR-23765: Update the message error when the name exists

pmPaulis committed:
solvin obs about remove the else

pmPaulis committed:
Merge pull request #8181 from ProcessMaker/feature/FOUR-23805

FOUR-23805: Update the text of modal to create a URL and validate the URL

caleeli committed:
Add test for updating case started script task status without logging errors

caleeli committed:
Implement check to skip non-user tasks in updateTaskStatus method

marcoAntonioNina committed:
change the order for filtering

gustavobascope committed:
fixed the add settings API when changing the settings from a bundle

caleeli committed:
Replicate issue: DOMDocument::loadXML(): Entity 'nbsp' not defined in Entity

pmPaulis committed:
FOUR-23895

caleeli committed:
Add tests for verify process endpoints

caleeli committed:
Implement HTML entity replacement for BPMN data in ProcessController

pmPaulis committed:
Merge pull request #8197 from ProcessMaker/feature/FOUR-23895

FOUR-23895: The SSO configurations are not showing all the information

caleeli committed:
Add test file process_with_html.bpmn

ryancooley committed:
ci[veracode]: add DAST scan to dev pushes

ryancooley committed:
Merge pull request #8228 from ProcessMaker/veracode-dast-essentials

ci[veracode]: add DAST scan to dev pushes

ryancooley committed:
ci[veracode]: update to new workflow version

ryancooley committed:
Merge pull request #8229 from ProcessMaker/veracode-fix

ci[veracode]: update to new workflow version

pmPaulis committed:
FOUR-24286

pmPaulis committed:
FOUR-24286: adding filter contains

pmPaulis committed:
Adding the comments about the change

eiresendez committed:
fix(sync-default-templates): handle GitHub fetch failures gracefully to avoid pipeline breakage

- Added timeout and warning log when fetching individual template files from GitHub fails.
- Replaced exception with `continue` to skip failed templates without stopping execution.
- Ensures that transient GitHub issues do not interrupt the entire CI/CD pipeline.

eiresendez committed:
add test for skipping failed template fetches and logging warnings

boliviacoca committed:
Merge pull request #8245 from ProcessMaker/task/FOUR-24430

FOUR-24430: CI Unit Test are given error SyncDefaultTemplates.php:79

nolanpro committed:
Update README.md

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

nolanpro committed:
Update cors.php

ryancooley committed:
Merge branch 'release-2025-spring' into develop

ryancooley committed:
Merge pull request #8199 from ProcessMaker/bugfix/FOUR-23865

FOUR-23865: Fix loading tasks in saved search with process with html entities

ryancooley committed:
Merge pull request #8233 from ProcessMaker/bugfix/FOUR-24286

FOUR-24286: Advanced filter on PROCESS in task saved search does not work

ryancooley committed:
Merge pull request #8177 from ProcessMaker/epic/FOUR-22580

FOUR-22580: [SUMMER 25] IFrame Whitelist Configs

ryancooley committed:
Update enterprise packages

ryancooley committed:
Update Nayra

ryancooley committed:
Update SSR executor

processmaker-bot committed:
Version 4.15.0+beta-1

ryancooley committed:
Update enterprise package

processmaker-bot committed:
Version 4.15.0+beta-1

pmPaulis committed:
FOUR-13976

pmPaulis committed:
Merge remote-tracking branch 'origin/develop' into bugfix/FOUR-13976-b

henryjonathanquispe committed:
FOUR-23828:It is possible to create Site name with blank spaces in IFrame white list

pmPaulis committed:
FOUR-24545

henryjonathanquispe committed:
FOUR-23828:It is possible to create Site name with blank spaces in IFrame white list

danloa committed:
Make AI env. variables accessibles to the UI

danloa committed:
Add ai config for genie timeout

caleeli committed:
Replicate issue FOUR-24548

caleeli committed:
Fix handling of empty non published nested screens

caleeli committed:
Add console command to clear screens cache

devmiguelangel committed:
update processmaker/docker-executor-php to version 1.4.0

caleeli committed:
Merge remote-tracking branch 'origin/pm-apps' into pm-apps-2

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

eiresendez committed:
fix(guided-template): override button width for deep-nested "btn-bx1" selector using ::v-deep

danloa committed:
Filter available variable in saved search

gustavobascope committed:
added a fix upgrade to change nullable the encrypted field

danloa committed:
Group logic to get variables when savedsearches are used

sanjacornelius committed:
Ensure only admin roles can assign super admins and all permissions

sanjacornelius committed:
Ensure only the super admin toggle is not visible for non admin users

sanjacornelius committed:
Backend validation to ensure only admin users can update super user permissions

ryancooley committed:
Update PLG package

ryancooley committed:
Merge pull request #8261 from ProcessMaker/bugfix/FOUR-23828-A

FOUR-23828: It is possible to create Site name with blank spaces in IFrame white list

ryancooley committed:
Merge pull request #8263 from ProcessMaker/bugfix/FOUR-24545

FOUR-24545

processmaker-bot committed:
Version 4.15.0-RC1

processmaker-bot committed:
Version 4.15.0

sanjacornelius committed:
Add tests for admin role assignments

marcoAntonioNina committed:
Validate upload avatar XSS

marcoAntonioNina committed:
remove logs

fagubla committed:
FOUR-24665:IFrame Whitelist recognize others protocol beside https

fagubla committed:
avoid urls with path

rodriquelca committed:
FOUR-24577: Threads of cases in parallel are reloaded automatically into a parent case

julceslauhub committed:
FOUR-24634 Business Logic Flaw Allows Admin API Token Generation

fagubla committed:
add validation URL in modal Edit

fagubla committed:
use mixin

ryancooley committed:
Merge pull request #8286 from ProcessMaker/sec/FOUR-24635

Restrict Super Admin Assignment to Admin Users Only

ryancooley committed:
Merge pull request #8293 from ProcessMaker/bugfix/FOUR-24634

FOUR-24634 Business Logic Flaw Allows Admin API Token Generation

ryancooley committed:
Merge pull request #8287 from ProcessMaker/bugfix/FOUR-24633

[CRITICAL] Insecure Avatar Upload Handling Leading to Stored XSS

ryancooley committed:
Merge pull request #8271 from ProcessMaker/FOUR-24570

FOUR-24570 scripts not able to rebuild

ryancooley committed:
Merge pull request #8268 from ProcessMaker/bugfix/FOUR-24259

Axios timeour after 60000ms in Flow Genie

ryancooley committed:
Merge pull request #8269 from ProcessMaker/bugfix/FOUR-24548

FOUR-24548 Fix screen rendering issue with empty nested screen

ryancooley committed:
Merge pull request #8191 from ProcessMaker/observation/FOUR-23820

observation/FOUR-23820 devLink > options are not synchronized

ryancooley committed:
Merge pull request #8193 from ProcessMaker/bugfix/FOUR-13976-b

FOUR-13976: Web entry link not redirecting to the screen when using default SSO SAML authentication

ryancooley committed:
Merge pull request #8281 from ProcessMaker/observation/FOUR-24603

observation/FOUR-24603 Server Error when using DevLink

ryancooley committed:
Merge pull request #8280 from ProcessMaker/bugfix/FOUR-24546

FOUR-24546L The columns available in a save search can be duplicated each time we enter the save search configuration.

ryancooley committed:
Merge pull request #8278 from ProcessMaker/bugfix/FOUR-24616

FOUR-24616: The "Save and Finish" button when creating a process from a guide template is distorted

ryancooley committed:
Merge pull request #8187 from ProcessMaker/FOUR-22013

FOUR-22013 Fix log of Case not Found when a script task is executed

ryancooley committed:
Merge pull request #8291 from ProcessMaker/bugfix/FOUR-24577_B

FOUR-24577: Threads of cases in parallel are reloaded automatically into a parent case

pmPaulis committed:
FOUR-24777: Migration error during upgrade

gproly committed:
FOUR-24684 It is possible to set an invalid URL in Iframe white list.
 ##Steps to Reproduce
Go to admin
Click on IFrame Whitelist
Add a new URL
Set a name
Set URL
Save the changes
Delete the url
Click on edit set a wrong url like an string or number
Save the changes

 ## Current Behavior
The edit whitelist does not validate the correct values

 ## Expected Behavior
Edit form in whitelist should validate correct values entered

 ## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-24684

gproly committed:
FOUR-24684 Add key white_list_frame.

pmPaulis committed:
Error with the upgrade

ryancooley committed:
Merge pull request #8301 from ProcessMaker/bugfix/FOUR-24684

FOUR-24684 It is possible to set an invalid URL in Iframe white list.

ryancooley committed:
Merge pull request #8288 from ProcessMaker/bugfix/FOUR-24665

FOUR-24665:IFrame Whitelist recognize others protocol beside https

ryancooley committed:
Update modeler & screen-builder

ryancooley committed:
Update enterprise packages

processmaker-bot committed:
Version 4.15.1+beta-1

henryjonathanquispe committed:
FOUR-23939: Task Section has an empty column

henryjonathanquispe committed:
FOUR-23939: Task Section has an empty column

pmPaulis committed:
FOUR-24522: Create Index to Improve Slow Query on cases_participated

marcoAntonioNina committed:
Merge branch 'develop' of github.com:ProcessMaker/processmaker into bugfix/FOUR-23812

ryancooley committed:
Merge pull request #8304 from ProcessMaker/bugfix/FOUR-23812

The variables are duplicated in Available columns in TASK and REQUEST Saved search

ryancooley committed:
Merge pull request #8303 from ProcessMaker/bugfix/FOUR-24522

FOUR-24522: Create Index to Improve Slow Query on cases_participated

ryancooley committed:
Merge pull request #8299 from ProcessMaker/bugfix/FOUR-24777

FOUR-24777: Migration error during upgrade with 2025_02_25_215239_encrypt_devlink_tokens

ryancooley committed:
Merge pull request #8302 from ProcessMaker/bugfix/FOUR-23939

Bugfix/four 23939

ryancooley committed:
Update enterprise package

processmaker-bot committed:
Version 4.15.1+beta-2

ryancooley committed:
Merge pull request #8306 from ProcessMaker/observation/FOUR-23820

FOUR-23820: devLink > options are not synchronized

ryancooley committed:
Update Enterprise package

processmaker-bot committed:
Version 4.15.1+beta-2

mcraeteisha committed:
Set minimum characters to 2

henryjonathanquispe committed:
FOUR-23617: When reloading the page, the completed status filter shows all clear

henryjonathanquispe committed:
FOUR-23617: When reloading the page, the completed status filter shows all clear

pmPaulis committed:
FOUR-24946

pmPaulis committed:
Adding test

henryjonathanquispe committed:
FOUR-24963: Cases and Tasks are not listed in launchpad when a saved search is added

sanjacornelius committed:
Implement is-admin property

fagubla committed:
FOUR-23812:The variables are duplicated in Available columns in TASK and REQUEST Saved search

ryancooley committed:
Merge pull request #8314 from ProcessMaker/bugfix/FOUR-23812

FOUR-23812:The variables are duplicated in Available columns in TASK and REQUEST Saved search

ryancooley committed:
Merge pull request #8309 from ProcessMaker/task/FOUR-24834

FOUR-24834 | Set Username Character Minimum to Two

ryancooley committed:
Update enterprise packages

processmaker-bot committed:
Version 4.15.1-RC1

sanjacornelius committed:
Remove troubleshooting code

processmaker-bot committed:
Version 4.15.1

pmPaulis committed:
FOUR-24986

fagubla committed:
FOUR-23812:The variables are duplicated in Available columns in TASK and REQUEST Saved search

ryancooley committed:
Merge pull request #8312 from ProcessMaker/feature/FOUR-24946

FOUR-24946: The API task needs to load the 'data' only if is necesary

ryancooley committed:
Merge pull request #8318 from ProcessMaker/bugfix/FOUR-24892

Fix: Ensure Ellipsis Menu is Displayed for Admin Users Without Default Project Permissions

ryancooley committed:
Merge pull request #8313 from ProcessMaker/bugfix/FOUR-24963

FOUR-24963: Cases and Tasks are not listed in launchpad when a saved search is added

ryancooley committed:
Merge pull request #8310 from ProcessMaker/bugfix/FOUR-23617

FOUR-23617: When reloading the page, the completed status filter show all clear

fagubla committed:
Update ProcessVariableController.php

ryancooley committed:
Update JS dependencies

ryancooley committed:
Update enterprise package

processmaker-bot committed:
Version 4.15.2+beta-1

pmPaulis committed:
Update some test

pmPaulis committed:
Merge pull request #8323 from ProcessMaker/feature/FOUR-24986

FOUR-24986: Code Integration with simdjson and uopz

henryjonathanquispe committed:
FOUR-25042: The Priority and Draft tab are moved to the Inbox, when you click the browser's reloaded this page

ryancooley committed:
Merge pull request #8322 from ProcessMaker/bugfix/FOUR-23812

FOUR-23812:The variables are duplicated in Available columns in TASK and REQUEST Saved search

ryancooley committed:
Update JS dependencies

ryancooley committed:
Update enterprise packages

processmaker-bot committed:
Version 4.15.2-RC1

processmaker-bot committed:
Version 4.15.2

gproly committed:
FOUR-24952 Updated and fixed observations.

gproly committed:
FOUR-24952 Fix missing file.

gproly committed:
FOUR-24952 Fix the missing function.

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

julceslauhub committed:
FOUR-25078 Response time in data connector resources has increased after the upgrade to version 4.14

gproly committed:
FOUR-25161 Implement SIMDJSON for json_encode
 ## Description
Implements support for https://github.com/awesomized/simdjson-plus-php-ext.

 ## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-25161

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

julceslauhub committed:
FOUR-25078 Response time in data connector resources has increased after the upgrade to version 4.14 - Unit test

julceslauhub committed:
FOUR-25078 Response time in data connector resources has increased after the upgrade to version 4.14 - Fix unit test

gproly committed:
FOUR-25161 Fix observations.

gproly committed:
FOUR-25161 Fix observations.

gproly committed:
FOUR-25161 Update documentation

gproly committed:
FOUR-25161 Fix The test documentation is completed.

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

caleeli committed:
Add metrics env variables

caleeli committed:
Disable store metrics for settings

caleeli committed:
Add scheduled command for clearing metrics with configurable intervals

nolanpro committed:
Merge pull request #8338 from ProcessMaker/bugfix/FOUR-25078

FOUR-25078 Response time in data connector resources has increased afer the upgrade to version 4.14

nolanpro committed:
Update enterprise packages

processmaker-bot committed:
Version 4.15.3+beta-1

pmPaulis committed:
Using json_decode in task Controller

pmPaulis committed:
Merge remote-tracking branch 'origin/develop' into epic/FOUR-24952

pmPaulis committed:
Removing the response_json_optimize

sanjacornelius committed:
Remove microsoft-graph dependency; Downgrade php-http/promise

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

pmPaulis committed:
Solving the observations

pmPaulis committed:
Adding notes in the PR

pmPaulis committed:
solving a unit test

pmPaulis committed:
Merge pull request #8341 from ProcessMaker/feature/FOUR-25161

FOUR-25161 Implement SIMDJSON for json_encode

pmPaulis committed:
removing the unused changes

caleeli committed:
Merge remote-tracking branch 'origin/develop' into FOUR-24338

danloa committed:
Add accesors for Message Events

danloa committed:
Merge pull request #8355 from ProcessMaker/task/FOUR-25207

Add accesors for Message Events

caleeli committed:
Clean app_name to be used for prometheus_namespace

processmaker-bot committed:
Version 4.15.3-RC1

sanjacornelius committed:
Remove unused graph import

sanjacornelius committed:
Revert and prevent composer.lock patch updates

pmPaulis committed:
CR observations, remove the json_optimize_encode

pmPaulis committed:
Solving the Unit test observations

danloa committed:
Merge pull request #8362 from ProcessMaker/epic/FOUR-24127

Add accesors for throw message events

processmaker-bot committed:
Version 4.15.3

sanjacornelius committed:
Merge branch 'develop' into task/FOUR-25182

nolanpro committed:
Merge pull request #8349 from ProcessMaker/FOUR-24338

FOUR-24338: Reduce metrics Memory usage

nolanpro committed:
Merge pull request #8324 from ProcessMaker/epic/FOUR-24952

FOUR-24952: JSON Optimization

nolanpro committed:
Merge pull request #8326 from ProcessMaker/bugfix/FOUR-25042

FOUR-25042: The Priority and Draft tab are moved to the Inbox, when you click the browser's reloaded this page

nolanpro committed:
Merge pull request #8352 from ProcessMaker/task/FOUR-25182

Move microsoft-graph Dependency to ABE Package and Restore Office 365 Functionality

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

nolanpro committed:
Update dependencies

processmaker-bot committed:
Version 4.15.4

processmaker-bot committed:
Version 4.15.4+beta-1

nolanpro committed:
Update dependencies

processmaker-bot committed:
Version 4.15.4+beta-1

processmaker-bot committed:
Version 4.15.4+beta-1

processmaker-bot committed:
Version 4.15.4+beta-1

danloa committed:
Remomve microsoft graph dependency

danloa committed:
Merge with develop

danloa committed:
Add library

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net

vladyrichter commented:
QA server K8S was successfully deployed https://ci-51a3650908.engk8s.processmaker.net
